Apps/CesiumViewer/CesiumViewer.js

```diff
@@ -42,12 +42,12 @@ define([
      */
     var endUserOptions = queryToObject(window.location.search.substring(1));
 
-    var imageryProvider;
-    if (endUserOptions.tmsImageryUrl) {
-        imageryProvider = createTileMapServiceImageryProvider({
-            url : endUserOptions.tmsImageryUrl
-        });
-    }
+
+    var imageryProvider = createTileMapServiceImageryProvider({
+        url : 'https://cesiumjs.org/blackmarble',
+        credit : 'Black Marble imagery courtesy NASA Earth Observatory',
+        flipXY : true // Only old gdal2tile.py generated tilesets need this flag.
+    });
```

Comment:
I'm sure you noticed this already, but these changes need to be reverted.

Reply:
I didn't notice. Those changes must have went in before I started working on this branch. Thanks.